### PR TITLE
RHOL-418 : Add support for multiple --eval parameters

### PR DESCRIPTION
**Pull request by odeac**
* changed the parser for --eval parameter to accept multiple instances
* change type of conf.eval to List[String] instead of String
* change the functions that take the fileName such that they take List[String] as input
* concatenate all the input files into one big string before executing it

Jira: https://rchain.atlassian.net/browse/RHOL-418

### CODE REVIEW

File: node/src/main/scala/coop/rchain/node/ReplClient.scala [hunk 1]
```diff
@@ -29,14 +29,24 @@ class GrpcReplService(host: String, port: Int) extends ReplService[Task] {
...
+    val (existing, missing) =
+      fileNames
+        .map(new File(_))
+        .partition(_.exists())
+
+    if (missing.isEmpty) {
+      val sources = existing.map(f => Source.fromFile(f))
+
+      try {
+        val content = sources.map(_.getLines().mkString("\n")).mkString("|\n")
```
Comment (guardbotmk3) [inline node/src/main/scala/coop/rchain/node/ReplClient.scala]: This is doubly the wrong approach.

A) We should not be merging the lines of the file. I'm certain scala has the ability to read the entire contents of a file.
B) We should send multiple RPC's

Comment (odeac) [reply node/src/main/scala/coop/rchain/node/ReplClient.scala]: A) As you can see on the left, this was the original implementation. I can fix it if needed

B) See my latest commit: 93b8bc4e

Comment (odeac) [reply node/src/main/scala/coop/rchain/node/ReplClient.scala]: for point (A) see 6ea6c06a

File: node/src/main/scala/coop/rchain/node/ReplClient.scala [hunk 2]
```diff
@@ -28,15 +25,20 @@ class GrpcReplService(host: String, port: Int) extends ReplService[Task] {
   def run(line: String): Task[String] = Task.delay {
     blockingStub.run(CmdRequest(line)).output
   }
+  def eval(fileNames: List[String]): Task[String] =
```
Comment (goral09) [inline node/src/main/scala/coop/rchain/node/ReplClient.scala]: Is this being used anywhere? I can't see it.

Comment (odeac) [reply node/src/main/scala/coop/rchain/node/ReplClient.scala]: Yes. It is called via the ReplService trait in node/.../repl.scala:39

**Pull request merged**